kedro-datasets/kedro_datasets/api/__init__.py

```diff
@@ -9,9 +9,8 @@
 import lazy_loader as lazy
 
 # https://github.com/pylint-dev/pylint/issues/4300#issuecomment-1043601901
-APIDataSet: type[APIDataset]
 APIDataset: Any
```

Comment:
Is there anything we can simplify or optimise in these `__init__.py` files now that the aliases are gone @deepyaman ?

Reply:
@astrojuanlu the `__init__.py` files are now back to how they were before all the aliasing.